entire_vds_pipeline.py

```diff
@@ -431,8 +431,12 @@ def export_to_elasticsearch(
     matched = sample_query.intersection(samples_in_vds)
     if len(matched) < len(sample_query) and args.ignore_extra_sample_ids_in_tables:
         logger.warning('Found only {0} out of {1} samples specified for ID remapping'.format(len(matched), len(sample_query)))
+        logger.info('Failed to find the following requested IDs for remapping: {}'.format(
```

Comment:
just pushed some style suggestions to https://github.com/macarthur-lab/hail-elasticsearch-pipelines/compare/master...style-edits 
let me know if you disagree on these